src/shared/components/Header/Header.less

```diff
@@ -1,7 +1,54 @@
 .Header {
-  background-color: #ff6666;
+  display: flex;
+  justify-content: space-between;
+  height: 60px;
+  background-color: #f0f0f0;
```

Comment:
variables should be preferred to plain values

Reply:
Agreed! I just need to figure out how to make them work with my webpack setup 😬